### DISCUSSION TITLE
第2节课作业(11班)

### DISCUSSION THREAD

**Discussion by keesh0410** [InternLM/Tutorial#108] [作业]
不忘初心加油加油！🐱

**Comment by horacehht** [comment]
https://blog.csdn.net/m0_50123831/article/details/135424296

**Comment by morning-start** [comment]
https://flowus.cn/morningstar/share/3ef9de41-839a-4562-b91b-f1ecbe9f5d22
【FlowUs 息流】轻松玩转书生·浦语大模型趣味Demo

**Comment by seifer08ms** [comment]
https://zhuanlan.zhihu.com/p/676380404

**Comment by chenyang126** [comment]
https://blog.csdn.net/weixin_50797351/article/details/135428272?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135428272%22%2C%22source%22%3A%22weixin_50797351%22%7D

**Comment by qianlian-mozi** [comment]
https://github.com/qianlian-mozi/internlm_practising_20240106/tree/main/Lesson_2_Homework

**Comment by blackxer** [comment]
https://zhuanlan.zhihu.com/p/676420543

**Comment by Shengshenlan** [comment]
https://github.com/Shengshenlan/tutorial/blob/master/homework/2/%E7%AC%AC%E4%BA%8C%E8%8A%82%E4%BD%9C%E4%B8%9A.md

**Comment by Everfighting** [comment]
https://github.com/Everfighting/InternLM_learning/blob/main/chapter2/chapter_2.md

**Comment by BarryYin** [comment]
第二节课作业：https://github.com/BarryYin/internLM/blob/main/%E7%AC%AC%E4%BA%8C%E8%8A%82%E4%BD%9C%E4%B8%9A.md

**Comment by Kongsea** [comment]
第二课作业：https://blog.csdn.net/deeepsea/article/details/135436430

**Comment by lvoooo** [comment]
https://github.com/lvoooo/internLM-learning/blob/main/2.work.md

**Comment by zhanghui-china** [comment]
基础作业：
[https://zhuanlan.zhihu.com/p/676369044](url)

进阶作业1：完成浦语·灵笔的图文理解及创作部署（需截图）
[https://zhuanlan.zhihu.com/p/675882989](url)

进阶作业2：完成 Lagent 工具调用 Demo 创作部署（需截图）
[https://zhuanlan.zhihu.com/p/676039643](url)

**Comment by LichKing-W** [comment]
https://zhuanlan.zhihu.com/p/676491000

**Comment by JustLike1** [comment]
https://blog.csdn.net/qq_45607888/article/details/135442829

**Comment by Vacili-f** [comment]
https://zhuanlan.zhihu.com/p/676540987

**Comment by codefd** [comment]
https://blog.csdn.net/googteo/article/details/135445056?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135445056%22%2C%22source%22%3A%22googteo%22%7D

**Comment by hellokid123** [comment]
https://github.com/hellokid123/internllm-notes/blob/main/task/task_of_lesson2/%E7%AC%AC%E4%BA%8C%E6%AC%A1%E8%AF%BE%E4%BD%9C%E4%B8%9A.md

**Comment by wwewwt** [comment]
https://github.com/wwewwt/-InternLM/wiki/2%E3%80%81%E4%BD%9C%E4%B8%9A

**Comment by wuyingxue1230** [comment]
https://zhuanlan.zhihu.com/p/676552277

**Comment by Fulin-Guo** [comment]
https://github.com/wdnmdm/Team_learning/blob/main/InternLM/%E7%AC%AC%E4%BA%8C%E8%AF%BE%E4%BD%9C%E4%B8%9A.md

**Comment by dukai289** [comment]
https://github.com/dukai289/Learning/blob/master/Learning_InternLM/Homework.md

**Comment by GMYelims** [comment]
https://blog.csdn.net/weixin_42112760/article/details/135451212?spm=1001.2014.3001.5501

**Comment by chenxiangzhen** [comment]
https://github.com/chenxiangzhen/InternLM-Tutorial/blob/main/%E7%AC%AC2%E8%8A%82.md

**Comment by moyitech** [comment]
https://blog.csdn.net/qq_30650051/article/details/135416872

**Comment by Unicornlyy** [comment]
http://t.csdnimg.cn/mUbVL

**Comment by Lyusijie** [comment]
https://blog.csdn.net/weixin_44986362/article/details/135445282?spm=1001.2014.3001.5501

**Comment by godlikeS97** [comment]
https://blog.csdn.net/godlikesun97/article/details/135492715?spm=1001.2014.3001.5502

**Comment by Tangent-90C** [comment]
https://github.com/Tangent-90C/public-note/tree/main/%E7%AC%AC%E4%BA%8C%E8%8A%82%E8%AF%BE%E7%9A%84%E4%BD%9C%E4%B8%9A

**Comment by keesh0410** [comment]
https://zhuanlan.zhihu.com/p/679128378

**Comment by Y-HuiMing-Y** [comment]
http://t.csdnimg.cn/PD8G7
